packages/common-sdk/src/web3/transactions/transactions-builder.ts

```diff
@@ -251,6 +255,11 @@ export class TransactionBuilder {
         (computeBudgetOption.priorityFeeLamports * MICROLAMPORTS_PER_LAMPORT) / computeLimit,
       );
       prependInstructions = [
+        SystemProgram.transfer({
```

Comment:
just curious but if the user is using an eclipse rpc, what happens here? im assuming that the same JTO addresses dont exist in eclipse right?

Reply:
I have the Eclipse jito tip set to 0 currently (since don't think jito tips exist on eclipse). It will just be an 0-transfer instruction